docs/cody/capabilities/supported-models.mdx

```diff
@@ -37,15 +37,17 @@ Cody uses a set of models for autocomplete which are suited for the low latency
 
 For all models, Cody allows up to **4,000 tokens of output**, which is approximately **500-600** lines of code.
 
-For Claude 3 Sonnet or Opus models users, Cody supports up to **30,000 tokens (~4,000 lines of code)** of user-defined context through `@-mention` syntax and **15,000 tokens** of continued chat history (user queries + system responses). All other models are currently capped at **7,000 tokens** of shared context between user-defined context and chat history.
+For Claude 3 Sonnet or Opus models, Cody supports up to **30,000 tokens (~4,000 lines of code)** of user-defined context through `@-mention` syntax and **15,000 tokens** of continued chat history (user queries + system responses). Chat history also includes context items that Cody has automatically retrieved.
+
+All other models are currently capped at **7,000 tokens** of shared context between user-defined context and chat history.
 
 Here's a detailed breakdown of the token limits by model:
 
 
 
 <Tabs>
   <Tab title="Free & Pro">
-|      **Model**      | **User Defined** | **Chat** **History** | **Output** |
+|      **Model**      | **Chat History** | **@-mentions**       | **Output** |
```

Comment:
If we are calling `@-mentions` as `user-defined context` above, maybe we should keep it as `user-defined context`, or update the section above?

Reply:
👍 I fixed this